src/code/z_fbdemo.c

```diff
@@ -1,9 +1,25 @@
+/**
+ * @file z_fbdemo.c
+ *
+ * This file implements an unused transition system that takes the current screen and converts it to large tiles.
+ *
+ * The static version will just display this tiled screen, while the dynamic version has the visual effect of `sucking`
+ * the tiled screen into a single point.
```

Comment:
"sucked" is in backticks like its referencing a variable or something, but it isnt right?
can maybe just say "visual effect of being sucked into a single point", or something